actions/container-ops/action.yml

```diff
@@ -2,14 +2,10 @@ name: 'Container operations'
 description: ' Build and Publish Docker image for applications packaged as docker artifact'
 
 inputs:
-  dockerfile:
```

Comment:
Why this parameter is removed?

Reply:
If we are going to have option to provide a specific Dockerfile to build and ignore other dockerfiles it makes sense to leave that parameter, but if the script will auto detect dockerfiles and build the image, then I fail to see why this parameter is useful?

https://github.com/htec-infra/gha-utils/pull/6#discussion_r795497599

Reply:
This workflow should work even in case that `app.conf` doesn't exist. So just imagine one dockerfile in repo and this action.

Reply:
From previous comments app.conf will be set as optiona while CONTAINER_REPO will be mandatory at the moment. Even if we do not supply removed parameter script will still fallback to default DIR/Dockerfile.
https://github.com/htec-infra/gha-utils/blob/fix/composite-actions/scripts/container_ops.sh#L23
That's the reason why I have removed that input from this composite action.

Only in one case this will fail and that's when CONTAINER_REPO is not present.